store/src/java/com/zimbra/cs/mailbox/calendar/ZAttendee.java

```diff
@@ -142,10 +142,10 @@ public ZAttendee(ZProperty prop) {
         setRole(prop.paramVal(ICalTok.ROLE, null));
         setPartStat(prop.paramVal(ICalTok.PARTSTAT, null));
 
-        String rsvpStr = prop.paramVal(ICalTok.RSVP, "FALSE");
-        boolean rsvp = false;
-        if (rsvpStr.equalsIgnoreCase("TRUE"))
-            rsvp = true;
+        String rsvpStr = prop.paramVal(ICalTok.RSVP, "TRUE");
```

Comment:
See https://tools.ietf.org/html/rfc2445#section-4.2.17 - the default value of RSVP is FALSE.
I think this change is too intrusive and may lead to bugs.  Presumably, the attendee information from the REPLY is at some point being applied to the ORGANIZER's calendar entry.  The code that does that change needs to consider what should be taken from the reply and what should not.